crates/primitives/src/state.rs

```diff
@@ -261,9 +261,13 @@ impl AccountInfo {
     }
 
     /// Returns account info without the code.
-    pub fn without_code(mut self) -> Self {
-        self.take_bytecode();
-        self
+    pub fn without_code(&self) -> Self {
```

Comment:
This is a breaking change. Can make a separate function call to do this if necessary

Reply:
went ahead and removed this